syntax/igorpro.vim

```diff
@@ -562,13 +562,14 @@ syn match   igorType      "^\s*\<wave\>"
 syn match   igorPreproc   "^#\(pragma\|define\|else\|elif\|endif\|ifdef\|ifndef\|if\|include\|undef\)"
 syn match   igorDelimiter "[(){}]\|\[\|\]"
 
-syn match   igorComment   "//.*"  contains=igorExtra
-syn match   igorExtra     "\(todo\|fixme\)"  contained
+syn match   igorComment   "//.*"  contains=igorTab,igorExtra
+syn match   igorExtra     "\(todo\|fixme\)" contained
+syn match   igorTab       "\t" contained
 
-syn region  igorFunctionFold  start="^\s*\(\|static\s\+\|threadsafe\s\+\|static\s\+threadsafe\s\+\|threadsafe\s\+static\s\+\)function\(\s\+\|\/\)" end="^\s*end\(\s\+\|$\)" fold transparent keepend contains=ALL
-syn region  igorMacroFold     start="^\s*\(macro\|window\|proc\)\s\+" end="^\s*\(end\|endmacro\)\(\s\+\|$\)" fold transparent keepend contains=ALL
-syn region  igorStructureFold start="^\s*\(\|static\s\+\)structure " end="^\s*endstructure" fold transparent keepend contains=ALL
-syn region  igorPictureFold start="^\s*\(\|static\s\+\)picture " end="^\s*end" fold transparent keepend contains=ALL
+syn region  igorFunctionFold  start="^\s*\(\|static\s\+\|threadsafe\s\+\|static\s\+threadsafe\s\+\|threadsafe\s\+static\s\+\)function\(\s\+\|\/\)" end="^\s*end\(\s\+\|$\)" fold transparent keepend contains=ALLBUT,igorTab,igorExtra
+syn region  igorMacroFold     start="^\s*\(macro\|window\|proc\)\s\+" end="^\s*\(end\|endmacro\)\(\s\+\|$\)" fold transparent keepend contains=ALLBUT,igorTab,igorExtra
+syn region  igorStructureFold start="^\s*\(\|static\s\+\)structure " end="^\s*endstructure" fold transparent keepend contains=ALLBUT,igorTab,igorExtra
+syn region  igorPictureFold start="^\s*\(\|static\s\+\)picture " end="^\s*end" fold transparent keepend contains=ALLBUT,igorTab,igorExtra
```

Comment:
@ukos-git Thx. Very nice idea!

A quick question. Why are you excluding igorExtra here?

Reply:
@ukos-git Good point. Works as expected here.